extension/index.googleFunction.js

```diff
@@ -22,11 +22,11 @@ exports.extensionTrigger = async (request, response) => {
     )
     if (paymentResult.success) {
```

Comment:
Didn't you remove this `success` flag?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/c8c8588ec91d3750e5a68597667c8777624eee87

Reply:
I still see you have `success` attr in the code. Check for all occurrences of it.

Reply:
sorry for missing
https://github.com/commercetools/commercetools-adyen-integration/pull/656/commits/23fef0bc40d6506b90fa7379dbd06236fc0d78ee

remove all success attribute in extension.